wandb/cli/cli.py

```diff
@@ -1953,4 +1956,5 @@ def verify(host):
         and large_post_success
         and url_success
     ):
+        print("Find detailed logs at: {}".format(os.path.join(tmp_dir, "wandb")))
```

Comment:
I guess it's ok, but curious why we're printing this twice...

Reply:
I'll drop this.